web/app/css/octopus.css

```diff
@@ -1,30 +1,34 @@
 @import 'styles.css';
 
 .octopus-graph {
-  padding: calc(var(--base-width) * 4);
+  max-width: 940px;
+  padding: calc(var(--base-width) * 2);
+  margin-left: auto;
+  margin-right: auto;
   background-color: #FAFAFA;
 
   & .octopus-body {
     background-color: white;
-    margin-bottom: calc(var(--base-width) * 3);
     border: 1px solid #F2F2F2;
+    padding: var(--base-width);
   }
 
   & .octopus-col {
     text-align: center;
-  }
-  & .resource-col {
-    padding: 32px;
+
+    & .neighbor {
+      height: 240px;
```

Comment:
This is super nit-picky, but I feel like a height of 240px is slightly too tall for the neighbor boxes, compared to the main box:

![image](https://user-images.githubusercontent.com/9226/45910824-50660080-bdc1-11e8-8463-a2b25951dfe5.png)

I'd prefer for the padding at the bottom of the neighbor box below the latency number be roughly proportional to the padding below the latency number in the main box.